2269/2/bad word dict

```diff
@@ -0,0 +1 @@
+х*й п**д еба* бл* на**й ебал х*есо еб* уе*ан п*зд хуй блять блядь пизд еба нахуй
```

Comment:
it's better to name files without spaces (use  _ or - instead).

Reply:
fixed